javaparser-core/src/main/java/com/github/javaparser/printer/PrettyPrinterConfiguration.java

```diff
@@ -82,6 +126,39 @@ public PrettyPrinterConfiguration setIndentSize(int size) {
         return this;
     }
 
+    /**
+     * Get the type of indent to produce.
+     */
+    public IndentType getIndentType() {
+        return indentType;
+    }
+
+    /**
+     * Set the type of indent to produce.
+     */
+    public PrettyPrinterConfiguration setIndentType(IndentType indentType) {
+        this.indentType = assertNotNull(indentType);
+        return this;
+    }
+
+    /**
+     * Get the tab width for pretty aligning.
+     */
+    public int getTabWidth() {
+        return tabWidth;
+    }
+
+    /**
+     * Set the tab width for pretty aligning.
+     */
+    public PrettyPrinterConfiguration setTabWidth(int tabWidth) {
+        if (tabWidth <= 0) {
```

Comment:
Yep, time to create that `assertPositive` you mentioned.

Reply:
Well, it was `assertNonNegative` for `indentSize` and is `assertPositive` for `tabWidth`.
Actually I had added `assertPositive` to `Util`, but changed it to your style.
Want me to add it back in?